commit 508085971164974adb1c2cd493c4b8056bc9293e
Author: John Levon <john.levon@joyent.com>
Date:   2019-09-02T09:52:02+00:00 (5 weeks ago)
    
    OS-7977 -fno-shrink-wrap shouldn't be specified for gcc4 build

diff --git a/usr/src/uts/Makefile.uts b/usr/src/uts/Makefile.uts
index e95700780a..791ebe8859 100644
--- a/usr/src/uts/Makefile.uts
+++ b/usr/src/uts/Makefile.uts
@@ -254,7 +254,9 @@ CFLAGS_uts_i386 += -_gcc8=-mindirect-branch-register
 # Ensure that the standard function prologue remains at the very start of a function,
 # so DTrace fbt will instrument the right place.
 #
-CFLAGS_uts_i386 = -_gcc=-fno-shrink-wrap
+CFLAGS_uts_i386 += -_gcc7=-fno-shrink-wrap
+CFLAGS_uts_i386 += -_gcc8=-fno-shrink-wrap
+CFLAGS_uts_i386 += -_gcc9=-fno-shrink-wrap
 
 CSTD = $(CSTD_GNU99)
 
